libopenage/console/console.cpp

```diff
@@ -131,6 +132,23 @@ void Console::interpret(const std::string &command) {
 			this->write(line.c_str());
 		}
 	}
+	else if (command.substr(0,3) ==  "set") {
+		std::size_t first_space = command.find(" ");
+		std::size_t second_space = command.find(" ", first_space+1);
+		if (first_space != std::string::npos && second_space != std::string::npos) {
+			std::string name = command.substr(first_space+1, second_space-first_space-1);
+			std::string value = command.substr(second_space+1, std::string::npos);
+			Engine::get().get_cvar_manager().set(name,value);
+		}
+	}
+	else if (command.substr(0,3) == "get") {
+		std::size_t first_space = command.find(" ");
+		if (first_space != std::string::npos) {
+			std::string name = command.substr(first_space+1, std::string::npos);
+			std::string value = Engine::get().get_cvar_manager().get(name);
+			this->write(value.c_str());
+		}
+	}
```

Comment:
it works for now, but we really should implement the interpreter in python. that way, arbitrary commands can be entered and the game controlled later. The commands for cvars within the python environment should be something like `openage.cvar.set("name", value)`, which can be abbrevated in the console by doing `import openage.cvar as cvar` in the `code.interact()`, if we wanna have the regular interactive python as default console.

Otherwise we can start some own interpreter parsing commands like you did here, which can then invoke the `code.interact()` by some command like `python`.